frontend/src/lib/candidate/components/input/InputField.type.ts

```diff
@@ -2,7 +2,11 @@ export type InputFieldProps<TValue extends AnswerProps['value']> = {
   /**
    * The question that is to be answered.
    */
-  question: QuestionProps;
+  questionId: string;
+  /**
+   * The options for the question that is to be answered.
+   */
+  questionOptions?: Array<AnswerOption>;
```

Comment:
And in the same vein, this could be just `options` with a comment: The answer options for a single or multiple choice input.